examples/gaussian_process/plot_gpr_dim_selection.py

```diff
@@ -0,0 +1,82 @@
+"""
+=========================================================
+Illustration of SelectDimensionKernel
+=========================================================
+
+A simple two-dimensional regression example computed using two
+different kernels:
+1. With a product of an RBF kernel and ExpSineSquared, where each is applied
+   to one feature.
+2. With one RBF kernel applied on both features.
+"""
+
+# Authors: Behzad Tabibian <me@btabibian.com>
+#
+# License: BSD 3 clause
+
+
+from sklearn.gaussian_process import GaussianProcessRegressor
+from sklearn.gaussian_process.kernels import RBF, ExpSineSquared
+from sklearn.gaussian_process.kernels import SelectDimensionKernel
+
+
+import numpy as np
```

Comment:
The convention is to keep external imports at the top followed by sklearn imports with a space in between.